quetz/testing/fixtures.py

```diff
@@ -40,8 +40,14 @@ def database_url(sqlite_url):
 
 
 @fixture
-def engine(database_url):
-    sql_echo = bool(os.environ.get("QUETZ_TEST_ECHO_SQL", False))
+def sql_echo():
+    """whether to activate SQL echo during the tests or not."""
+    return False
```

Comment:
Is this on purpose? Why not use the env var here?

Reply:
it allows to override it using this fixture, see the modification in `engine`: it defaults sql_echo to the env var value

Reply:
I find it quite confusing tbh. Do we really need it?